src/PsTool/Common.cs

```diff
@@ -2819,15 +2819,21 @@ public static string MakeSingleCSS(string fullPath, string targetFileName)
 					{

```

Comment:
We can include testcase for this CSS function.

Reply:
I updated the unit test in the latest Pull request. changes (see below).